unbounce/sb-sample-extension
# FEAT/typed-manifest

https://unbounce.atlassian.net/browse/CN-7378

## What is this change?

- Sets up using a typed manifest for local development.

## Why are we making this change?

Typing the manifest will cut down on the number of type errors we've been dealing with from third-party apps.

## Background Info / Screenshots:

<img width="405" alt="image" src="https://user-images.githubusercontent.com/20562946/179864739-8e6cfc23-eeec-45dc-b648-8602d96d2955.png">


## Testing Notes

Enable the feature flag `features/typedManifest` in the Smart Builder

1. `npm run start` and `npm run serve`
2. Open the apps sidebar in the Smart Builder
3. Add 8081 as the port number
=> The Hello World app should be loaded in
